src/pages/[platform]/build-a-backend/existing-resources/cli/index.mdx

```diff
@@ -0,0 +1,996 @@
+
+
+### Clean up resources
+
+Once you're finished experimenting with this demo app, we recommend deleting the backend resources to avoid incurring unexpected costs. You can do this by running the following command in the root folder of the app.
+
+```bash showLineNumbers={false}
+amplify delete
+
+```
+If you would like to expand this demo app into a production-ready app, you may need to add additional resources, such as authorization and storage. Refer to the [Build & connect backend section](/[platform]/build-a-backend/) for guides on how to add and connect other backend resources.
+
+## Connect web app to existing AWS resources
+
+This guide will walk you through connecting a new React web app to the AWS resources created with Amplify for an existing React app. If you don't already have an existing app, you can follow this [React tutorial](https://docs.amplify.aws/react/start/getting-started/introduction/) to create a to-do app that uses Amplify Auth, API, and Hosting resources.
+
+Before you begin, you will need:
+
+* An existing React app
+* An AWS account: If you don't already have an account, follow the [Setting Up Your AWS Environment](https://aws.amazon.com/getting-started/guides/setup-environment/) tutorial for a quick overview.
+* [Install](https://docs.amplify.aws/cli/start/install/) and configure the Amplify CLI.
+* A text editor. For this guide, we will use VS Code, but you can use your preferred IDE.
+
+### Find the AWS backend details
+Before connecting the backend resources to our new app, we first need to locate the details of the AWS environment provisioned for the existing app.
+
+**Step 1:** In your existing app, open the file `<Your-App>/amplify/team-provider-info.json` .
+
+<img src="/images/existing-resources/team-provider-info-web-cli.png" alt="The team-provider-info.json file within the file directory of the Amplify app." width="450" />
+
+**Step 2:** In the `team-provider-info.json` file, note the following:
+
+1. The environment you want to use 
+2. The `AmplifyAppId` for the required environment
+
+{/* cSpell:disable */}
+<img src="/images/existing-resources/dev-amplifyappid-web-cli.png" alt="The environment and AmplifyAppId in team-provider-info.json file." width="450" />
+{/* cSpell:enable */}
+
+### Create the React app
+Now that we have gathered the necessary backend details, we can start building out the new React app.
+
+**Step 1:** Create a React app by running the following command in your terminal.
+
+```bash showLineNumbers={false}
+npx create-react-app react-amplify-connect
+```
+
+**Step 2:** Open the newly created React app using VS Code by running the following commands in your terminal. 
+
+```
+cd react-amplify-connect
+code . -r
+```
+
+![Open the created app using VS Code.](/images/existing-resources/app-vscode-web-cli.png)
```

Comment:
Again it is not clear to me what I should be looking at in this image - may consider resizing so can see menu better if expected to review.  Otherwise I'm not sure it is needed.